pennylane/ops/qutrit/__init__.py

```diff
@@ -31,6 +31,8 @@
     "QutritUnitary",
     "TShift",
     "TClock",
+    "TAdd",
+    "TSWAP",
```

Comment:
`TSWAP` is in fact Hermitian. While it can be used as an observable, I don't see the benefit for the same reason that `SWAP` isn't an observable. I'm not sure how useful it is, and it's a bit more restrictive in its use as any circuit potentially using it as an observable would need to have multiple wires.

Reply:
Cool, we can leave it as an operation for now 👍🏼, and revisit it in a later PR